src/e2e/resources/data/AdminAccountsPageE2ETest.json

```diff
@@ -57,7 +63,7 @@
       "name": "Teammates Instr2",
       "email": "AAccounts.instr2@gmail.tmt",
       "role": "Co-owner",
-      "isDisplayedToStudents": false,
+      "isDisplayedToStudents": true,
```

Comment:
As mentioned above, I wanted to test that the admin could in fact delete an instructor. Because the built-in requirement is that a visible instructor is left, I changed the original default instructor to be visible, so the other one would be deleted successfully.

Also, as far as I understand, it makes more sense for the instructor to be visible, as an admin can't even delete instructors if there aren't any visible ones left (making it seem to me that it's assumed there needs to be one visible instructor of a course)